typescript/cli/src/commands/status.ts

```diff
@@ -19,12 +19,11 @@ export const statusCommand: CommandModuleWithWriteContext<
       description: 'Dispatch transaction hash',
     },
   },
-  handler: async ({ context, origin, destination, id, relay, dispatchTx }) => {
```

Comment:
Considering that the destination var is unused maybe we should remove it from the command options too so that users won't try to set it and then see unexpected results because the value is not used